src/settings.ts

```diff
@@ -157,9 +157,9 @@ export namespace Settings {
         public: true,
         advanced: true,
         options: {
-          no: "No",
-          yes: "Yes",
-          yesone: "Yes, one archive",
+          no: i18n.__("settings.zipArchive.no"),
+          yes: i18n.__("settings.zipArchive.yes"),
+          yesone: i18n.__("settings.zipArchive.yesone"),
```

Comment:
Please change all translations for the dropdown values like this: 
```
no: i18n.__("settings.zipArchive.value.no"),
yes: i18n.__("settings.zipArchive.value.yes"),
 ...
```

Reply:
Sure, done.